common/data/base/houseBase.ts

```diff
@@ -0,0 +1,38 @@
+import { fetchAPI, getObjectsTotalCount, STRAPI_BASE_URL } from '../dataHelpers';
+import { canGetById } from './canGetById';
+
+export const houseBase = {
+    async getObject(id: string): Promise<HouseObject> {
+        return canGetById.getObject(id, '/house');
+    },
+    async getObjectsPolygonsByRange(from: string, to: string, filterName: string) {
+        const totalCount = await getObjectsTotalCount(`${STRAPI_BASE_URL}/house`);
+        return (await fetchAPI(`${STRAPI_BASE_URL
+        }/house?populate=borders&filters[${filterName}][$gte]=${from}&filters[${filterName}][$lte]=${to}&pagination[pageSize]=${totalCount}`))
+            .data.map((x) => x.attributes.borders?.coordinates);
```

Comment:
I just saw that in strapi those coordinates are srtings, can you cast them to numbers here?

Reply:
No need! I just asked Olya to fix it in strapi

Reply:
Also Olya is going to switch places of coordinates, so we don't need to do it here anymore